src/btree/bt_vrfy.c

```diff
@@ -418,8 +418,8 @@ __verify_addr_ts(WT_SESSION_IMPL *session, WT_REF *ref, WT_CELL_UNPACK_ADDR *unp
  *     Our job is to check logical relationships in the page and in the tree.
  */
 static int
-__verify_tree(
-  WT_SESSION_IMPL *session, WT_REF *ref, WT_CELL_UNPACK_ADDR *addr_unpack, WT_VSTUFF *vs)
+__verify_tree(WT_SESSION_IMPL *session, WT_REF *ref, WT_CELL_UNPACK_ADDR *addr_unpack,
+  uint64_t parent_write_gen, WT_VSTUFF *vs)
```

Comment:
Does this need a new parameter for `parent_write_gen`? It is derived from a parameter that is already passed in from both callers in the same way. Could it just be extracted within the function instead of passed as a new parameter?

Reply:
Thanks for the finding. Yes, the parameter is no longer required.